examples/model_selection/plot_multi_metric_evaluation.py

```diff
@@ -0,0 +1,85 @@
+"""Demonstration of multi-metric evaluation on cross_val_score and GridSearchCV
+
+Multiple metric grid search (or random search) can be done by setting the
+``scoring`` parameter to a list of metric scorer names or a dict mapping the
+scorer names to the scorer callables.
+
+The scores of all the scorers are available in the ``cv_results_`` dict at keys
+ending in ``'_<scorer_name>'`` (``'mean_test_precision'``,
+``'rank_test_precision'``, etc...)
+
+The ``best_estimator_``, ``best_index_``, ``best_score_`` and ``best_params_``
+correspond to the scorer (key) that is set to the ``refit`` attribute.
+"""
+
+# Author: Raghav RV <rvraghav93@gmail.com>
+# License: BSD
+
+import numpy as np
+from matplotlib import pyplot as plt
+
+from sklearn.datasets import make_hastie_10_2
+from sklearn.model_selection import GridSearchCV
+from sklearn.metrics import make_scorer
+from sklearn.metrics import f1_score
+from sklearn.tree import DecisionTreeClassifier
+
+
+print(__doc__)
+
+
+X, y = make_hastie_10_2(n_samples=8000, random_state=42)
+
+# The scorers can be a standard scorer referenced by its name or one a
+# callable-like returned from make_scorer
+scoring = {'AUC Score': 'roc_auc', 'F1 Score': make_scorer(f1_score)}
+
+# Multiple metric GridSearchCV, best_* attributes are exposed for the scorer
```

Comment:
The point is not that those are exposed the point is that we refit with those parameters. Also maybe say "We set refit='AUC Score' which means we will refit an estimator on the whole training set using the parameter setting that has the best cross-validated AUC score.